summoner.cabal

```diff
@@ -47,7 +47,7 @@ library
   other-modules:       Paths_summoner
   autogen-modules:     Paths_summoner
 
-  build-depends:       base-noprelude >= 4.10 && < 5
+  build-depends:       base-noprelude >= 4.12 && < 5
```

Comment:
We are not going to give up the support of the `8.2.2` and `8.4.3` yet so no need to upgrade the lower bound. The cost of supporting those versions is very low (we already have everything working in there) but the usability on different versions is very valuable!